samples/quickstart.js

```diff
@@ -13,7 +13,7 @@
 
 'use strict';
 
-async function main() {
+async function main(parent) {
```

Comment:
I might make this `projectId`, and do that thing we usually do with samples where a few lines down we have:

```
// const projectId = ....
```